crm_meeting_commercial_partner/models/res_partner.py

```diff
@@ -8,21 +8,22 @@
 class ResPartner(models.Model):
     _inherit = "res.partner"
 
-    meeting_count = fields.Integer(compute='_compute_meeting_count')
+    meeting_count = fields.Integer(
+        compute='_compute_meeting_count_commercial_partner')
 
     @api.multi
-    def _compute_meeting_count(self):
-        super()._compute_meeting_count()
+    def _compute_meeting_count_commercial_partner(self):
+        event_model = self.env['calendar.event']
         for partner in self:
-            partners = partner + partner.child_ids
-            meetings = self.env['calendar.event'].search(
-                [('partner_ids', 'in', partners.ids)])
-            partner.meeting_count = len(meetings)
+            partner.meeting_count = event_model.search_count(
```

Comment:
Use `read_group` instead for total performance boost. @mreficent has done it before in for example: https://github.com/OCA/contract/commit/402370f1cbe5cde6b484200f87596e9d803a36af#diff-3dc0d52ea9449a93459ef5b7fc36a060R22

Reply:
Thanks @pedrobaeza ! It is much faster now! :)